scripts/gh_pr.sh

```
@@ -0,0 +1,149 @@
+    git fetch origin pull/$pr_id/head:pr-$pr_id
+    git checkout pr-$pr_id
+}
+
+function test_pr() {
+    local pr_id=$1
+    if [ -z "$pr_id" ]; then
+        echo "Usage: $0 test <pr_number>"
+        exit 1
+    fi
+
+    local original_branch=$(git rev-parse --abbrev-ref HEAD)
```

Shellcheck (0.11.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)